cli/azd/pkg/tools/git/git.go

```diff
@@ -143,7 +143,7 @@ func (cli *gitCli) InitRepo(ctx context.Context, repositoryPath string) error {
 	}
 
 	// Set initial branch to main
-	runArgs = exec.NewRunArgs("git", "-C", repositoryPath, "branch", "-M", "main")
+	runArgs = exec.NewRunArgs("git", "-C", repositoryPath, "branch", "-b", "main")
```

Comment:
Good catch. This seems to work on newer versions of git, but perhaps on older versions you can't run git branch -M <branch.>